src/js/actions/guides.js

```diff
@@ -66,13 +75,20 @@ define(function (require, exports) {
                 range: "guide",
                 index: 1,
                 count: -1
-            },
-            getOpts = {
-                failOnMissingProperty: true
             };
 
-        // FIXME: Should we reverse these?
-        return descriptor.getPropertiesRange(docRef, rangeOpts, _guideProperties, getOpts);
+        var requiredPromise = descriptor.getPropertiesRange(docRef, rangeOpts,
```

Comment:
There were enough differences that I didn't want to attempt to separate them at this point (extensions, start indices being different, etc.)